index.css

```diff
@@ -0,0 +1,84 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+#main {
+  background-color: silver;
+}
+
+#header {
+  padding-top: 5%;
+  padding-left: 40%;
+}
+
+#header p {
+  padding: 2rem;
+}
+
+#img-div {
+  padding: 5rem;
+  text-align: center;
+  display: block;
+  margin: 2rem;
+  background-color: #fff;
+}
+
+#img-div img {
+  width: 100%;
+}
+
+#tribute-info {
+  margin-left: 20%;
+  margin-right: 20%;
+}
+
+#tribute-info h2 {
+  margin: 1rem;
+  text-align: center;
+}
+
+#tribute-info li {
+  padding-top: 1em;
+}
+
+h3 {
+  padding-bottom: 5rem;
+}
+
+#main h3 {
+  margin-left: 20%;
+  margin-top: 2rem;
+}
+
+#main h3 a {
+  text-decoration: none;
+}
+
+figcaption {
+  margin-top: 1rem;
+}
```

Comment:
Kindly avoid unnecessary styles (styling elements that are not there in your HTML) This makes your code clean and more readable.